ptf_tests/common/lib/__init__.py

```diff
@@ -1,3 +1,19 @@
+# Copyright (c) 2022 Intel Corporation.
+#

```

Comment:
Please add the following to after the Copyright line in all files

`SPDX-License-Identifier: Apache-2.0`

See example in this PR: https://github.com/ipdk-io/ovs/pull/52/files

Reply:
Sure.